examples/scanner/scanner.ino

```diff
@@ -8,143 +8,113 @@
  */
 
 /**
- * Channel scanner and Continuous Carrier Wave Output
+ * Channel scanner and continuous carrier wave output
+ * 
+ * There are 126 channels available, 1MHz wide, starting at 2400MHz. This example will scan through all 126 channels, 
+ * listening for 128 microseconds on each channel for a carrier wave. The more carrier waves detected during the channel sweep, the 
+ * higher the hexadecimal value displayed under that channel on the serial monitor. The channel numbers are displayed in hexadecimal in
+ * a header two lines tall, where 0x00 is on the left and 0x7D is on the right.
  *
- * Example to detect interference on the various channels available.
- * This is a good diagnostic tool to check whether you're picking a
- * good channel for your application.
+ * While any 2.4GHz device can trigger this carrier wave detection, a second NRF24L01+ device can be configured to output a constant 
+ * carrier wave by sending the character "g" over the serial monitor. Output power and channel can be adjusted in the #defines below.
  *
- * Run this sketch on two devices. On one device, start CCW output by sending a 'g'
- * character over Serial. The other device scanning should detect the output of the sending
- * device on the given channel. Adjust channel and output power of CCW below.
- *
- * Inspired by cpixip.
- * See http://arduino.cc/forum/index.php/topic,54795.0.html
- */
+ * Originally inspired by cpixip,
+ * See https://forum.arduino.cc/t/poor-mans-2-4-ghz-scanner/54846/39
+*/
 
 #include "RF24.h"
-#include "printf.h"
-
-//
-// Hardware configuration
-//
 
-// Set up nRF24L01 radio on SPI bus plus pins 7 & 8
+#define CARRIER_POWER RF24_PA_LOW  //can be RF24_PA_MIN, RF24_PA_LOW, RF24_PA_HIGH, RF24_PA_MAX
+#define CARRIER_CHANNEL 40         //0-125
 
-RF24 radio(7, 8);
+// Set up nRF24L01 radio on SPI bus plus pins 9 & 10 for CE and CSN
+RF24 radio(5, 6);
 
-//
 // Channel info
-//
-
 const uint8_t num_channels = 126;
-uint8_t values[num_channels];
+uint8_t values[num_channels];  //number of times a value is detected on each channel during the scan
 
-//
-// Setup
-//
+const int num_reps = 100;  //number of times to scan through channels
+bool constCarrierMode = 0;
 
 void setup(void) {
-  //
-  // Print preamble
-  //
 
+  // Print preamble
   Serial.begin(115200);
-  printf_begin();
-  Serial.println(F("\n\rRF24/examples/scanner/"));
+  while (!Serial)
+    ;
+  Serial.println(F("rRF24/examples/scanner/"));
 
-  //
   // Setup and configure rf radio
-  //
+  if(!radio.begin())
+  {
```

Comment:
I believe this would've been adjusted by clang-format (which is now used in the Arduino IDE v2.x):
```suggestion
  if (!radio.begin()) {
```
We have a copy of the clang-format config in this lib's examples folder to ease style conformity with the newer Arduino IDE style.